src/js/store/products/reducer.ts

```diff
@@ -24,39 +26,51 @@ const defaultState = {
 
 const reducer = (
   products: ProductsState = defaultState,
-  action: ProductsAction | LogoutAction,
+  action: ProductsAction | ObjectsAction | LogoutAction,
```

Comment:
Still using `ProductsAction` here?

Reply:
There is still one custom `ProductsAction`, for syncing the repos.